example-config.edn

```diff
@@ -12,7 +12,7 @@
  ; Jira username (might be something like "flastname").
  :assignee #env USER
  ; Type of issue to create.
- :issue-type "Task"
```

Comment:
there is code that uses this value as `issue-type`: https://github.com/rwstauner/jirazzz/blob/main/jirazzz#L209-L217
so changing the spelling here without changing that code means it will never find an issue type from the createmeta response.